### PR TITLE
FixaMinGata cobrand changes

### PR DESCRIPTION
**Pull request by joesiltberg**
Misc. FixaMinGata-specific configurations from the last couple of months.

### CODE REVIEW

File: web/cobrands/fixamingata/js.js [hunk 1]
```diff
@@ -1 +1,20 @@
 fixmystreet.inspect_form_no_scroll_on_load = 1;
+
+// Chrome ignores autocomplete="off" on the title input,
+// and incorrectly autocompletes it with the user's email address.
+// For now we'll reset the title to empty if it contains
+// an email address when the user has selected a category.
+// Hopefully we can get rid of this eventually if Chrome changes
+// its behaviour.
+fixmystreet.fixChromeAutocomplete = function() {
+    let title = document.getElementById("form_title");
```
Comment (dracos) [inline web/cobrands/fixamingata/js.js]: Could this be var

File: web/cobrands/fixamingata/js.js [hunk 2]
```diff
@@ -1 +1,20 @@
...
+
+    if (title) {
+	if (title.value == "" ||
+	    /^[a-zA-Z0-9.!#$%&'*+\/=?^_`{|}~-]+@(?:\S{1,63})$/.test(title.value)) {
+            title.value = "";
+        }
+    }
+}
```
Comment (dracos) [inline web/cobrands/fixamingata/js.js]: Semicolon at end of this line

**Comment by joesiltberg** [comment]
Thanks for the feedback @dracos ! The PR is updated.

**Comment by dracos** [comment]
Great, thanks!

**Pull request merged**